frontend/PoliticsApp/News/Views/NewsView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  NewsView.swift
+//  PoliticsApp
+//
+//  Created by Ronald Jabouin on 11/18/22.
+//
+
+import SwiftUI
+
+struct NewsTabView: View {
+    
+    @StateObject var articlesNewsVM = ArticleNewsViewModel()
+    @State var searchText = ""
+    
+    var body: some View {
+                
+        NavigationView{
+            ArticleListView(articles: articles)
+                .overlay(overlawView)
+            //whenever the selected category chnages, the data is reloaded
+                .task(id: articlesNewsVM.fetchTaskToken, loadTask)
+                .refreshable(action: refreshTask)
+                .navigationTitle(articlesNewsVM.fetchTaskToken.category.text)
```

Comment:
not sure if this is necessary, changing the title to be whatever the category is. This should be a new field or indicated by something else (if you change the categories then the category field should be an indication)